dialog.js

```diff
@@ -204,7 +204,11 @@ async function setCertDisplayFields(result) {
       document.getElementById('certimg').innerHTML = "✅";
     }
   } else {
-    document.getElementById('certimg').innerHTML = "❌";
+    if (warning == null && location != null) {
```

Comment:
Invert this if chain, i.e.:

```
if (warning == null && location != null) {
  ⌛
} else if (correctContentType) {
  ✅
} else {
  ❌
}
```

Ditto setDisplayFields below.

Reply:
Done.